src/webots/wren/WbWrenAbstractResizeManipulator.cpp

```diff
@@ -136,6 +136,7 @@ void WbWrenAbstractResizeManipulator::initializeHandlesEntities(bool resize) {
 
   for (int i = 0; i < mNumberOfHandles; ++i) {
     WrRenderable *renderable = wr_renderable_new();
+    wr_renderable_set_scene_culling(renderable, false);
```

Comment:
Why is this needed?

Reply:
It is for consistency with the transform manipulator.